monero-outreach-docs/translations/es/PSA/files/wallet/wallet_es.sla

```diff
@@ -60,19 +64,98 @@
-            <trail LINESP="13.2"/>
+        <PAGEOBJECT OwnPage="0" PTYPE="4" XPOS="129.0008" YPOS="380.0016" WIDTH="273.6" HEIGHT="360" RADRECT="0" FRTYPE="0" CLIPEDIT="0" PWIDTH="0.432" PCOLOR="None" PCOLOR2="None" TXTFILLSH="90" COLUMNS="1" COLGAP="0" NAMEDLST="" SHADE="100" SHADE2="100" GRTYP="0" ROT="0" PLINEART="1" PLINEEND="0" PLINEJOIN="0" LINESP="13.2" LOCALSCX="1" LOCALSCY="1" LOCALX="0" LOCALY="0" PICART="1" PLTSHOW="0" BASEOF="0" textPathType="0" textPathFlipped="0" FLIPPEDH="0" FLIPPEDV="0" IFONT="Adobe Caslon Pro Regular" ISIZE="11" SCALETYPE="1" RATIO="1" PRINTABLE="1" ANNOTATION="0" ANNAME="" TEXTFLOWMODE="0" TEXTFLOW="0" TEXTFLOW2="0" TEXTFLOW3="0" AUTOTEXT="0" EXTRA="0" TEXTRA="0" BEXTRA="0" REXTRA="0" FLOP="0" PFILE="" PFILE2="" PFILE3="" PRFILE="" EPROF="" IRENDER="1" EMBEDDED="1" LOCK="0" LOCKR="0" REVERS="0" TransValue="0" TransValueS="0" TransBlend="0" TransBlendS="0" isTableItem="0" TopLine="0" LeftLine="0" RightLine="0" BottomLine="0" isGroupControl="0" NUMDASH="0" DASHS="" DASHOFF="0" NUMPO="16" POCOOR="0 0 0 0 273.6 0 273.6 0 273.6 0 273.6 0 273.6 360 273.6 360 273.6 360 273.6 360 0 360 0 360 0 360 0 360 0 0 0 0 " NUMCO="16" COCOOR="0 0 0 0 273.6 0 273.6 0 273.6 0 273.6 0 273.6 360 273.6 360 273.6 360 273.6 360 0 360 0 360 0 360 0 360 0 0 0 0 " NUMGROUP="0" GROUPS="" startArrowIndex="0" endArrowIndex="0" OnMasterPage="" ImageClip="" ImageRes="1" Pagenumber="0" isInline="0" fillRule="1" doOverprint="0" gXpos="0" gYpos="0" gWidth="0" gHeight="0" LAYER="0" BOOKMARK="0" NEXTITEM="-1" BACKITEM="-1">
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Tu monedero de Monero (XMR) es la caja fuerte que contiene tus XMR y los mantiene seguros. Si quieres asegurarte de mantenerlo así, existen buenas prácticas que quisieras seguir para mantener tus monederos lo "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="más seguros posible."/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Los monederos ‘calientes’ son monederos que se conectan "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="al internet, mientras que los monederos en ‘frío’ no. Los monederos en frío generalmente son más seguros pero "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="vienen con un sacrificio a conveniencia del quien los usa."/>
```

Comment:
"vienen con un sacrificio a la conveniencia." That mean they sacrifice convenience for safeness.

Reply:
I know it could be both. Will change it either way :).

Reply:
done.